packages/components/src/Inject/Inject.md

```diff
@@ -1,7 +1,94 @@
-### **INJECT**
+# **INJECT**
 
 With the Inject components you can instanciate any component anywhere. This allows a great flexibility in your component design.
 
+The concept is to let you add `slot` in your component so from the outside a user may
+inject in a controlled way some other components.
+
+# How to customize components as a user
+
+For components you can pass props like this one:
+
+```js
+    const props = {
+        components: {
+            'before-something': [
```

Comment:
could use some more information geared toward the component user like "before-something" is the slot name in which you wish to inject a component, you can find those in the component documentation.